types/api/readonly.d.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * ReadOnly API
+ */
+export interface ReadOnly {
+  /**
+   * Set or toggle read-only state
+   *
+   * @param {Boolean|undefined} state - set or toggle state
+   * @returns {Boolean} current value
+   */
+  toggleReadOnly: (state?: boolean) => boolean;
```

Comment:
it is better to set by default false rather than undefined

Reply:
My goal is to toggle the current value when state is undefined, then return the new value. Does this make sense?
https://github.com/codex-team/editor.js/pull/1035/files#diff-f1ccc28f8e2fb341fa79ad3ac36a0a16R36

Reply:
Oh, I got it. Yes, it won't make sense, when clicking twice with the false value :)